### ISSUE TITLE
TypeError: string indices must be integers

### ISSUE THREAD

**Issue by PerfectHawk** [samueldy/mark-canvas-announcements-as-read#1]
Hi Samuel,

so when I run the script, I get the following error:
... , line 54, in <module>
course_id = course["id"]
TypeError: string indices must be integers

How do I fix this?

**Comment by samueldy** [comment]
Not sure. I've never encountered that before. Might be a malformed JSON response where `course` is just a string. What is the output if you add a `print(courses.json())` just before line 53?

I'll push up a new branch for you to clone/test.

Edit: branch is `handle-missing-course-id`.

**Comment by PerfectHawk** [comment]
After adding that line it says {'errors':[ {'message': 'Invalid access token.'}]}.
So since I'm on windows, I took my home directory to be my user folder in the C: drive. Is that correct? I can try creating a new token as well an seeing if that will fix the issue. I'll edit this comment when I do

edit: So I see where my issue was. When I created the .instructure.json file, I put the code {"token": "<YOUR_TOKEN_HERE"} and only replaced the "Your_Token_here" with my token, but you needed to replace everything. It was my error. 

Thanks for your help!
